### PR TITLE
rMS proximal operator 

### PR DESCRIPTION
**Pull request by NickLuiken**
Pull request for the rMS proximal operator and the rMS tutorial

### CODE REVIEW

**Review by mrava87 — COMMENTED** [review]
@NickLuiken thanks for adding this, looks great.

I made some changes and also added a few comments where I was unsure. Please either reply here and I can fix it, or make sure to `git pull origin dev` before making the changes and pushing them to avoid conflicts with my changes

File: pyproximal/proximal/rMS.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import numpy as np
+
+from pyproximal.ProxOperator import _check_tau
+from pyproximal import ProxOperator
+
+
+def _l2(x, thresh):
```
[the remark this answers is not in the document]
Comment (NickLuiken) [reply pyproximal/proximal/rMS.py]: No it is the proximal of the l2 norm. Docstring has been changed

File: pyproximal/proximal/rMS.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    r"""scaling.
```
[the remark this answers is not in the document]
Comment (NickLuiken) [reply pyproximal/proximal/rMS.py]: Added more descriptive text

File: pyproximal/proximal/rMS.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+    Applies soft thresholding to vector ``x - g``.
```
[the remark this answers is not in the document]
Comment (NickLuiken) [reply pyproximal/proximal/rMS.py]: Done

File: pyproximal/proximal/rMS.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+    Parameters
+    ----------
+    x : :obj:`numpy.ndarray`
+        Vector
+    thresh : :obj:`float`
+        Threshold
```
[the remark this answers is not in the document]
Comment (NickLuiken) [reply pyproximal/proximal/rMS.py]: No, it is a regularization parameter and I changed the docstring

File: pyproximal/proximal/rMS.py [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+    .. [1] Strekalovskiy, E., and D. Cremers, 2014, Real-time minimization of the piecewise smooth
+            Mumford-Shah functional: European Conference on Computer Vision, 127–141.
+
+    """
+    def __init__(self, sigma=1., kappa=1., g=None):
+        super().__init__(None, False)
+        self.sigma = sigma
+        self.kappa = kappa
+        # self.g = g
```
[the remark this answers is not in the document]
Comment (NickLuiken) [reply pyproximal/proximal/rMS.py]: Not needed but was accidentally copied from another code. Removed now.

**Pull request merged**